conjur/init/init_controller.py

```diff
@@ -71,9 +103,10 @@ def get_server_certificate(self):
         sys.stdout.write("\nTo verify this certificate, it is recommended to run the following " \
                          "command on the Conjur server:\n" \
                          "openssl x509 -fingerprint -noout -in ~conjur/etc/ssl/conjur.pem\n\n")
-        trust_certificate = input("Trust this certificate? [no]: ")
+        trust_certificate = input("Trust this certificate? [no]: ").strip()
         if trust_certificate.lower() != 'yes':
-            raise ValueError("You decided not to trust the certificate")
+            logging.debug("Invalid input. Acceptable input includes: 'YES', 'yes' ")
```

Comment:
`no` is not invalid input.
I think this log is redundant anyway.

Reply:
removing